docs/release-notes/release-notes-0.19.0.md

```diff
@@ -63,6 +63,9 @@
 * [Fixed a bug](https://github.com/lightningnetwork/lnd/pull/9322) that caused
     estimateroutefee to ignore the default payment timeout.
 
+* [Make deletion of alias edge atomic](
```

Comment:
This is not a deletion, but upgrading to regular edges, IIUC. Can this be rephrased here and in PR title?